student.rb

```diff
@@ -1,6 +1,8 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
```

Comment:
- [ ] In the student class there should be a method that serves as the 'belongs-to' side (a student belongs to a classroom). please have a look at [this article ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)with examples